### PR TITLE
PE : placeholder for future addition

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Streamlink recently added a mdstrm plugin, so now every mediastre.am channel can be backed using free.fullspeed.tv, once updates will be made.

### CODE REVIEW

**Review by MapGuy11 — APPROVED** [review]

File: streams/pe.m3u [hunk 1]
```diff
@@ -155,6 +155,8 @@ http://cdnhd.iblups.com/hls/F87ppt1YAT.m3u8
 http://ott.streann.com:8080/loadbalancer/services/public/channels/59ce7f292cdc7ba015a93b82/playlist.m3u8
 #EXTINF:-1 tvg-id="LaRiberena.pe" status="error",La Ribereña (Pucallpa) (480p) [Not 24/7]
 https://inliveserver.com:1936/19020/19020/playlist.m3u8
+#EXTINF:-1 tvg-id="Latina.pe",Latina (720p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.latina.pe/tvenvivo
```
Comment (freearhey) [inline streams/pe.m3u:159]: Not working at the moment:

image

**Review by freearhey — DISMISSED** [review]

File: streams/pe.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/pe.m3u:159]: Yes, because I don't think @dct-infra pushed the last Streamlink changes to his repo yet.

**Comment by MapGuy11** [comment]
I accidentally marked it for review @LaneSh4d0w please change that.

**Review by its0x08 — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
He's got the latest changes, and since it's geo-locked to Peru only, it won't work basically.

**Pull request closed without merging**